libs/sdk-core/src/breez_services.rs

```diff
@@ -1185,12 +1202,12 @@ impl BreezServices {
             .target(env_logger::Target::Pipe(target_log_file))
             .parse_filters(
                 r#"
-                info,
+                debug,
                 breez_sdk_core::input_parser=warn,
                 breez_sdk_core::backup=info,
                 breez_sdk_core::persist::reverseswap=info,
                 breez_sdk_core::reverseswap=info,
-                gl_client=warn,
+                gl_client=debug,
```

Comment:
Did you mean to include log filter changes?

Reply:
Good catch! Actually I didn't but I find myself always change that when I need to debug, I think we can push this change as it enables logging the signer and gl-client node logs.
What do you think?

Reply:
👍  just making sure